### PR TITLE
chore(ecocredit): ecocedit cli audit updates

### PR DESCRIPTION
**Pull request by aleem1314**
## Description

Ref: #715

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/ecocredit/client/basket/query.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func QueryBasketsCmd() *cobra.Command {
 		Short: "Retrieves all baskets",
 		Long:  "Retrieves all baskets currently in state, with optional pagination",
 		Example: strings.TrimSpace(`regen q ecocredit baskets
-	regen q ecocredit baskets --pagination.offset 1 --pagination.limit 10`),
+	regen q ecocredit baskets --offset 1 --limit 10`),
```
Comment (ryanchristo) [inline x/ecocredit/client/basket/query.go:50]: I'm not sure we want to use `strings.TrimSpace` when defining `Example`. Would be nice to remove here and to use the same format as the examples below.

File: x/ecocredit/client/basket/query.go [hunk 2]
```diff
@@ -79,10 +80,10 @@ func QueryBasketBalanceCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "basket-balance [basket-denom] [batch-denom]",
 		Short: "Retrieves the balance of a credit batch in the basket",
-		Long: strings.TrimSpace(`Retrieves the balance of a credit batch in the basket
-Example:
-		$regen q ecocredit basket-balance BASKET C01-20210101-20220101-001
-		`),
+		Long:  strings.TrimSpace("Retrieves the balance of a credit batch in the basket"),
```
Comment (ryanchristo) [inline x/ecocredit/client/basket/query.go:83]: Same as above. I don't think we need to use `strings.TrimSpace` here.

File: x/ecocredit/client/tx.go [hunk 3]
```diff
@@ -266,8 +280,9 @@ Parameters:
                                 "start_date":       "1990-01-01",
                                 "end_date":         "1995-10-31",
                                 "project_jurisdiction": "AB-CDE FG1 345",
+								issuer: "regen1elq7ys34gpkj3jyvqee0h6yk4h9wsfxmgqelsw"
```
Comment (ryanchristo) [inline x/ecocredit/client/tx.go:283]:
```suggestion
                                "project_jurisdiction": "AB-CDE FG1 345",
                                "issuer": "regen1elq7ys34gpkj3jyvqee0h6yk4h9wsfxmgqelsw"
```

**Review by ryanchristo — APPROVED** [review]
Looks good to me. Just a few nits.

File: x/ecocredit/client/basket/tx.go [hunk 4]
```diff
@@ -204,22 +204,23 @@ Parameters:
...
-			{
-				"batch_denom": "C01-001-20210101-20220101-001",
-				"amount": "10.5"
-			}
-		]
 		`),
+		Example: `
+regen tx ecocredit put-in-basket eco.uC.NCT credits_json_file
+
+Where credits_json_file contains:
```
Comment (technicallyty) [inline x/ecocredit/client/basket/tx.go:211]:
```suggestion
regen tx ecocredit put-in-basket eco.uC.NCT credits.json

Where the credits.json file contains:
```

File: x/ecocredit/client/tx.go [hunk 5]
```diff
@@ -549,7 +574,11 @@ func TxCreateProject() *cobra.Command {
 		project-jurisdiction: the jurisdiction of the project (see documentation for proper project-jurisdiction formats).
 		metadata: any arbitrary metadata attached to the project.
 		Flags:
-		reference-id: proect reference id
+		reference-id: project reference id
+		`,
+		Example: `
+regen tx ecocredit create-project C01 AA-BB 12345 metadata
+regen tx ecocredit create-project C01 AA-BB 12345 metadata --reference-id R01
```
Comment (technicallyty) [inline x/ecocredit/client/tx.go:581]:
```suggestion
regen tx ecocredit create-project C01 "AA-BB 12345" metadata
regen tx ecocredit create-project C01 "AA-BB 12345" metadata --reference-id R01
```

**Pull request merged**